container/common/fsHandler.go

```diff
@@ -92,10 +92,11 @@ func (fh *realFsHandler) update() error {
 	fh.lastUpdate = time.Now()
 	if fh.rootfs != "" && rootErr == nil {
 		fh.usage.InodeUsage = rootUsage.Inodes
-		fh.usage.TotalUsageBytes = rootUsage.Bytes + extraUsage.Bytes
+		fh.usage.TotalUsageBytes = rootUsage.Bytes
 	}
 	if fh.extraDir != "" && extraErr == nil {
 		fh.usage.BaseUsageBytes = rootUsage.Bytes
```

Comment:
It looks like this line (fh.usage.BaseUsageBytes = rootUsage.Bytes) should be moved up to the previous if statement.  Not sure how we messed that up :).  Feel free to do that here, or i'll merge this the way it is.

Reply:
@dashpole 
Addressed.